server/public/styles.css

```diff
@@ -24,3 +24,7 @@ body {
   left: 50%;
   transform: translateY(-50%) translateX(-50%);
 }
+
+.new-block-formatting-context {
```

Comment:
Instead of creating this one-off class, include an inline `style` in your `PageHeader` component.